.golangci.yaml

```diff
@@ -11,6 +11,10 @@ issues:
     - path: '(.+)_test\.go'
       linters:
         - govet
+    - linters: [revive]
+      text: var-naming  # do not demand that `Id` always be spelled `ID`; messes with autogenerated code
+    - linters: [stylecheck]
+      text: ST1003  # do not demand that `Id` always be spelled `ID`; messes with autogenerated code
```

Comment:
It does disable the whole rule. ID is the most prominent clash between the linter and standard CamelCase, but I think there are others even in our existing .yaml. Maybe the word JSON? Not sure.

The fact that the rule is inherently partially applied in the first place (ID must be capitalized, as does JSON, but not ETH or SHA because they're not on the linter's list of acronyms. And I don't know what the rule is for RPC, for example) is what makes it bad, IMO.